### PR TITLE
Add b2s method on pkg/disk/type_bsd.go

### LINKED ISSUE #5035: Can't build minio on Mac
## Expected Behavior
Minio should build on Mac

## Current Behavior
 Minio can't be build

## Context
The b2s method was removed by this commit: https://github.com/minio/minio/commit/88938340b367c0dd222430692fbe1f014fd7b0e2#diff-ce06b3e78086ce45cd73136d438a5fee

But it's still in use on https://github.com/minio/minio/blob/master/pkg/disk/type_bsd.go#L23

## Your Environment
go version go1.9 darwin/amd64

### PR DESCRIPTION
**Pull request by jmaitrehenry**
## Description
Add method b2s for building minio on Mac

## Motivation and Context
Fixes #5035 

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by balamurugana** [comment]
@jmaitrehenry Please find travis-ci error i.e. formatting error.

**Comment by jmaitrehenry** [comment]
@balamurugana sorry for that, it's now fixed

File: pkg/disk/type_bsd.go [hunk 1]
```diff
@@ -22,3 +22,11 @@ package disk
 func getFSType(fstype [16]int8) string {
 	return b2s(fstype[:])
 }
+
+func b2s(bs []int8) string {
```
Comment (balamurugana) [inline pkg/disk/type_bsd.go]: There is no reason to keep this it as separate function.  This code needs to be inside `getFSType()`.  Sorry for the late catch.

**Review by balamurugana — DISMISSED** [review]

**Comment by jmaitrehenry** [comment]
@balamurugana fixed

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**